lib/core/versions/latest/Did.ts

```diff
@@ -161,7 +167,40 @@ export default class Did {
     return initialStateValue;
   }
 
+  private static constructCreateOperationFromEncodedJcs (initialStateEncodedJcs: string): CreateOperation {
+    // Initial state should be in the format base64url(JCS(initialState))
+    const initialStateDecodedJcs = Encoder.decodeAsString(initialStateEncodedJcs);
+    let initialStateObject;
+    try {
+      initialStateObject = JSON.parse(initialStateDecodedJcs);
+    } catch {
+      throw new SidetreeError(ErrorCode.DidInitialStateJcsIsNotJosn, 'long form initial state should be encoded jcs');
```

Comment:
Probably nit, but convention is that error message is a sentence. ie. Capital first letter, and ends with a full stop.